shell.sh

```diff
@@ -6,7 +6,7 @@ fi
 
 
 if [ $1 == "build" ]; then 
-    bazel build --repo_env=CC=clang //src:shell
+    bazel build --config=clang_config //shell:shell
```

Comment:
I have:
```
$ ./shell.sh build
INFO: Analyzed target //shell:shell (6 packages loaded, 33 targets configured).
INFO: Found 1 target...
ERROR: path-to-repo/architecture/shell/BUILD:3:11: undeclared inclusion(s) in rule '//shell:shell':
this rule is missing dependency declarations for the following files included by 'shell/src/commands/command.cpp':
  '/usr/lib/llvm-11/lib/clang/11.0.1/include/stddef.h'
  '/usr/lib/llvm-11/lib/clang/11.0.1/include/__stddef_max_align_t.h'
  '/usr/lib/llvm-11/lib/clang/11.0.1/include/stdarg.h'
  '/usr/lib/llvm-11/lib/clang/11.0.1/include/stdint.h'
In file included from shell/src/commands/command.cpp:2:
./shell/src/commands/command.h:31:35: warning: unused parameter 'args' [-Wunused-parameter]
    void execute(const Arguments& args, IState& state, IStreams& stream) override {}
                                  ^
./shell/src/commands/command.h:31:49: warning: unused parameter 'state' [-Wunused-parameter]
    void execute(const Arguments& args, IState& state, IStreams& stream) override {}
                                                ^
./shell/src/commands/command.h:31:66: warning: unused parameter 'stream' [-Wunused-parameter]
    void execute(const Arguments& args, IState& state, IStreams& stream) override {}
                                                                 ^
3 warnings generated.
Target //shell:shell failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 2,479s, Critical Path: 2,06s
INFO: 0 processes.
FAILED: Build did NOT complete successfully
```

Reply:
This PR reverts your tomorrow's

Reply:
This commit with old line gives me:
```
$ ./shell.sh build
ERROR: Skipping '//src:shell': no such package 'src': BUILD file not found in any of the following directories. Add a BUILD file to a directory to mark it as a package.
 - path-to-repo/architecture/src
WARNING: Target pattern parsing failed.
ERROR: no such package 'src': BUILD file not found in any of the following directories. Add a BUILD file to a directory to mark it as a package.
 - /home/radium/Documents/csc_study/2022_1/architecture/src
INFO: Elapsed time: 0,149s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
```

Should I mix them?

Reply:
btw, have you set your clang version in toolchain/clang_include_directories.bzl ? Crutch from previous realization :(

Reply:
should work
```
bazel build --repo_env=CC=clang //shell:shell
```